charge.go

```diff
@@ -91,6 +92,15 @@ type FraudDetails struct {
 	StripeReport FraudReport `json:"stripe_report"`
 }
 
+// Outcome is the charge's outcome that details whether a payment
+// was accepted and why.
+type Outcome struct {
```

Comment:
What do you think about naming this `ChargeOutcome` instead? That feels a little less ambiguous to me.